src/components/home/vacations-card.tsx

```diff
@@ -143,11 +149,59 @@ const VacationsCard = () => {
     fetchVacationsRequests();
   }, []);
 
+  /**
+   * Initialize logged in person's data.
+   */
+  const getPersonVacations = async () => {
+    if (persons.length) {
+      setLoading(true);
+      if (loggedInPerson || config.person.id) setLoading(false);
+      {
+        const fetchedPersons = await personsApi.listPersons({ active: true });
+        setPersons(fetchedPersons);
+      }
+    }
+  };
+
+  useMemo(() => {
+    if (!persons) {
+      getPersonVacations();
+    }
+  }, [persons]);
+
+  /**
+   * display persons vacation days
+   */
+  const renderVacations = (person: Person | undefined) => {
```

Comment:
Update JSDocs and maybe rename function to renderVacationDays, like it's described in the docs. Person parameter is used in the function and @param is missing from the docs.

Reply:
Added @param person to docs.